pkg/storj/encryption.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storj
+
+// EncryptionScheme is the scheme and parameters used for encryption
+type EncryptionScheme struct {
+	Algorithm      EncryptionAlgorithm
+	EncryptedKey   EncryptedPrivateKey
```

Comment:
Note that we currently generate a different random encryption key for each segment and store it in the segment's metadata.

Reply:
I'll move it to Segment, and we change it when the model changes.